client/src/App.jsx

```diff
@@ -52,15 +64,23 @@ class App extends React.Component {
 
   render () {
     window.addEventListener('click', function (e) {
+      let element;
+      if (e.target.parentNode === null) {
+        element = e.target;
+      } else {
+        element = e.target.parentNode;
+      }
       let time = new Date().toLocaleString();
       let body = {
         time: time,
-        module: findId(e.target.parentNode),
-        element: e.target.outerHTML
+        element: e.target.outerHTML,
+        widget: findId(element)
       };
-      console.log('this is your click event', body);
 
-      // create axios post to /interactions here
+      axios.post(`${localhost}/interactions`, body)
+        .then(() => console.log('posted to interactions'))
```

Comment:
Hey! Don't think this console log is needed anymore, yeah! I liked the one showing the component being clicked on and its details. Even that one isn't needed.

Reply:
✅